src/test/frontend.test.js

```diff
@@ -0,0 +1,27 @@
+const timeout = process.env.SLOWMO ? 30000 : 10000;
+const fs = require('fs');
+beforeAll(async () => {
+
+  await page.goto('file:///home/keshav/image-sequencer/examples/index.html', {waitUntil: 'domcontentloaded'});
```

Comment:
Please change the path here.I am working on it to make it relative path as suggested by @HarshKhandeparkar